src/main.c

```diff
@@ -460,6 +465,160 @@ void WINAPI EventCallback(PEVENT_RECORD ev)
     }
 }
 
+static void BluetoothHciEventCallback(PEVENT_RECORD ev)
+{
+    struct INTERFACE* Iface;
+    PROPERTY_DATA_DESCRIPTOR Desc;
+    TDHSTATUS Err;
+    UINT8 Type;
+    UINT32 DataLen;
+    PBYTE Data = NULL;
+    BOOL IsSend = FALSE;
+    ULARGE_INTEGER TimeStamp;
+
+    if (!IsEqualGUID(&ev->EventHeader.ProviderId, &BthPortCapId) ||
+        (ev->EventHeader.EventDescriptor.Id != tidHci && ev->EventHeader.EventDescriptor.Id != tidHciRaw)) {
+        return;
+    }
+
+    if (!Pass2) {
+        Iface = GetInterface(0);
```

Comment:
I'd expect at least a code comment explaining the use of the special value 0 here. How do you know this won't collide with a real LowerIfIndex?

Reply:
I'll have to do some more research to get a good explanation. I just figured out that is what is needed through reverse engineering.